packages/flutter/lib/src/material/magnifier.dart

```diff
@@ -310,12 +316,35 @@ class Magnifier extends StatelessWidget {
   /// the background.
   final Color filmColor;
 
-  /// The shadows for this [Magnifier].
+  /// A list of shadows cast by the [Magnifier].
```

Comment:
Any value in using a macro here for deduplicating the docs on CupertinoMagnifier.shadows? If they're not subtly different.

Reply:
they're actually more than just subtly different, they're sort of opposites of each other (at least in terms of defaults).

(Also and this is maybe more of a dubious point but so much of what I fixed in this PR was overzealous use of templates making the docs actively worse that I'm kind of reluctant to use them at all!)